libs/live-connection/project.json

```diff
@@ -1,30 +1,39 @@
 {
+    "name": "live-connection",
+    "$schema": "../../node_modules/nx/schemas/project-schema.json",
     "projectType": "library",
     "sourceRoot": "libs/live-connection/src",
     "targets": {
         "lint": {
-            "executor": "@nrwl/linter:eslint",
+            "executor": "@nx/linter:eslint",
             "options": {
-                "lintFilePatterns": ["libs/live-connection/**/*.ts"]
+                "lintFilePatterns": [
+                    "libs/live-connection/**/*.ts"
+                ]
             }
         },
         "test": {
-            "executor": "@nrwl/workspace:run-commands",
+            "executor": "nx:run-commands",
             "options": {
-                "command": "npx vitest --run --passWithNoTests",
+                "command": "branchProtectionPrompt vitest --run --passWithNoTests",
```

Comment:
What is `branchProtectionPrompt`?

Reply:
I'm puzzled myself, not sure where that came from. But it is already fixed, when I don't think you reviewed the latest...